admin/class-admin-apple-notice.php

```diff
@@ -237,7 +233,7 @@ private static function add_user_meta( $user_id, $value ) {
 		if ( defined( 'WPCOM_IS_VIP_ENV' ) && true === WPCOM_IS_VIP_ENV ) {
 			return update_user_attribute( $user_id, self::KEY, $values );
 		} else {
-			return update_user_meta( $user_id, self::KEY, $values );
+			return update_user_meta( $user_id, self::KEY, $values ); // phpcs:ignore WordPress.VIP.RestrictedFunctions.user_meta_update_user_meta
```

Comment:
Since this is a plugin that is meant to work on VIP and Non-VIP sites, it might be worth disabling this restricted function sniff in `phpcs.xml` so that you do not need to iungore it each time you user it.

Reply:
I'd rather have it pop the error and suppress it intentionally with the comment, just to be safe. It's also a signal to VIP that we know what we're doing :)